assets/src/edit-story/components/form/color/colorPreview.js

```diff
@@ -56,15 +57,10 @@ const TextualPreview = styled.div`
   text-align: center;
 `;
 
-function ColorPreview({
-  onChange,
-  hasGradient,
-  hasOpacity,
-  isMultiple,
-  value,
-  label,
-}) {
-  const previewStyle = getPreviewStyle(isMultiple ? null : value);
+function ColorPreview({ onChange, hasGradient, hasOpacity, value, label }) {
+  const isMultiple = value === MULTIPLE_VALUE;
```

Comment:
A general note: Should `isMultiple` should also check for multiple elements actually being selected?

Reply:
Right now, `value === MULTIPLE_VALUE` can only be sent for multi-elements selection. That's the key reason for a the `MULTIPLE_VALUE` no longer being just an empty string.